internal/service/ec2/vpc_ipam_preview_next_cidr_test.go

```diff
@@ -0,0 +1,113 @@
+package ec2_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+)
+
+func TestAccVPCIpamPreviewNextCidr_ipv4Basic(t *testing.T) {
+	resourceName := "aws_vpc_ipam_preview_next_cidr.test"
+	netmaskLength := "28"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acctest.PreCheck(t) },
```

Comment:
After merge of #22391, you can rebase and then

```suggestion
		PreCheck:     func() { acctest.PreCheck(t); testAccIPAMPreCheck(t) },
```

Reply:
context: https://github.com/hashicorp/terraform-provider-aws/pull/22391

Reply:
Resolved in ac1190b26e4b2e3ac4a54b367a514a2d22e39184